top-level getting started portal page

rpetrusha opened the pull request:
# Top-level C# Getting Started topic

## Summary

The portal page for C# Getting Started topics

## Details

Because the TOC is collapsible, if a user is using the TOC for navigation, it looks as if there is no child content because the parent has yet to be written. So I thought I'd write it. This topic mirrors only written content; I haven't included boilerplate topics marked with a wrench.

## Suggested Reviewers

@billwagner @stevehoag @mairaw @kendrahavens

Commit:
top-level getting started portal page

saldana commented:
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/csharp/getting-started/index.md](https://github.com/rpetrusha/docs/blob/starting/docs/csharp/getting-started/index.md) | :x:Error | | [Details](#user-content-docs/csharp/getting-started/index.md)

<a name="docs/csharp/getting-started/index.md"></a>
### [docs/csharp/getting-started/index.md](https://github.com/rpetrusha/docs/blob/starting/docs/csharp/getting-started/index.md)
 - **Line Unspecified**: **[Error]** Author ronpet cannot be recognized in GitHub.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C11%5C16%5Ca3bda507-3390-de91-8f7f-26f90f4e5fc8%5CPullRequest%5C201611162051119805-1252%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

stevehoag commented on docs/csharp/getting-started/index.md:
```diff
@@ -13,16 +13,27 @@ ms.assetid: b77c7263-7cbf-4729-9626-8fbc3f5f14d9
+
+   This topic shows you how to create and run a simple Hello World application with Visual Studio code and .NET Core.
+
+- [Building a C# Hello World application with .NET Core in Visual Studio 2017 RC](with-visual-studio-2017.md)
```
Shouldn't this be the VS 2015 topic?

stevehoag commented on docs/csharp/getting-started/index.md:
```diff
@@ -2,7 +2,7 @@
 description: Getting Started with C#
 keywords: C#, Getting Started, Acquisition, Install
-author: dotnet-bot
+author: ronpet
```
rpetrusha - ms.author is ronpet

stevehoag commented on docs/csharp/getting-started/index.md:
```diff
@@ -13,16 +13,27 @@ ms.assetid: b77c7263-7cbf-4729-9626-8fbc3f5f14d9
+
+- [Getting started with Visual Studio Code](with-visual-studio-code.md)
+
+   Visual Studio code is a programming editor for Windows, Linux, and macOS that supports IntelliSense (code completion) and debugging.
```
code should be capitalized

stevehoag commented on docs/csharp/getting-started/index.md:
```diff
@@ -13,16 +13,27 @@ ms.assetid: b77c7263-7cbf-4729-9626-8fbc3f5f14d9
+
+   Visual Studio code is a programming editor for Windows, Linux, and macOS that supports IntelliSense (code completion) and debugging.
+
+   This topic shows you how to create and run a simple Hello World application with Visual Studio code and .NET Core.
```
Code

Commit:
Corrected errors, added library topic, fixed typo in another topic

rpetrusha commented:
@stevehoag I've addressed all of your comments.
I also forgot (???) to include the class library topic, so I've added that.
Finally, I noticed a typo in one of the 2015 topics. I've corrected that and included it in this PR.

stevehoag commented:
LGTM. Pending a successful build, :shipit:

saldana commented:
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/csharp/getting-started/index.md](https://github.com/rpetrusha/docs/blob/starting/docs/csharp/getting-started/index.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/getting-started/index?branch=pr-en-us-1252) | [Details](#user-content-docs/csharp/getting-started/index.md)
[docs/csharp/getting-started/publishing-with-visual-studio.md](https://github.com/rpetrusha/docs/blob/starting/docs/csharp/getting-started/publishing-with-visual-studio.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/getting-started/publishing-with-visual-studio?branch=pr-en-us-1252) | [Details](#user-content-docs/csharp/getting-started/publishing-with-visual-studio.md)

<a name="docs/csharp/getting-started/index.md"></a>
### [docs/csharp/getting-started/index.md](https://github.com/rpetrusha/docs/blob/starting/docs/csharp/getting-started/index.md)
 - **Line 29**: **[Warning]** Invalid file link:(~/docs/csharp/getting-started/library-with-visual-studio-2017.md).


For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C11%5C16%5Ca3bda507-3390-de91-8f7f-26f90f4e5fc8%5CPullRequest%5C201611162141084292-1252%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.
